src/test/scala/com/microsoft/hyperspace/index/LogicalPlanSerDeTests.scala

```diff
@@ -63,16 +67,43 @@ class LogicalPlanSerDeTests extends SparkFunSuite with SparkInvolvedSuite {
     singleTablePlan = Project(
       Seq(c1, c2, c3),
       Filter(And(IsNotNull(c3), EqualTo(c3, Literal("facebook"))), scanNode))
+
+    val csvRelation = HadoopFsRelation(
+      tableLocation,
+      new StructType(),
+      tableSchema,
+      None,
+      new CSVFileFormat,
+      Map.empty)(spark)
+    val csvScanNode = LogicalRelation(csvRelation, Seq(c1, c2, c3, c4), None, isStreaming = false)
```

Comment:
you are not updating `var csvScanNode`? Also, why not just create this at the test method?

Reply:
I think this is why the test is failing.

Reply:
thanks, fixed the tests